client/COPY.json

```
@@ -166,6 +166,8 @@
   "CAVC_EXTENSION_REQUEST_DENY_SUCCESS_TITLE": "You have denied extension",
   "CAVC_EXTENSION_REQUEST_DENY_SUCCESS_DETAIL": "This task can be completed to distribute to a judge for further processing",
 
+  "ADD_CAVC_DATES_TITLE": "Add Court dates",
```

Do you want to through "CAVC" in here somewhere like?
```suggestion
  "ADD_CAVC_DATES_TITLE": "Add CAVC mandate dates",
```
You can ask Ann-Marie when you demo it.

---

Yea dunno, going off of the [designs](https://www.figma.com/file/8noUqRbvCJzdpwkSNCb8lZ/CAVC-Remand-Designs?node-id=882%3A625)

---

Holding off on this one as I believe it's best to go off what design has mocked up unless @araposo-tistatech would like it changed